src/ngx_http_lua_api.c

```diff
@@ -74,4 +86,123 @@ ngx_http_lua_add_package_preload(ngx_conf_t *cf, const char *package,
     return NGX_OK;
 }
 
+
+ngx_shm_zone_t *
+ngx_http_lua_shared_memory_add(ngx_conf_t *cf,
+                               ngx_str_t *name,
+                               size_t size,
+                               void *tag)
```

Comment:
Style: better avoid unnecessary line breaks above unless 78 columns are to be exceeded otherwise.